src/software/ai/hl/stp/tactic/stop/stop_fsm.cpp

```diff
@@ -2,7 +2,7 @@
 
 void StopFSM::updateStop(const Update& event)
 {
-    event.common.set_primitive(createStopPrimitive(coast));
```

Comment:
Please also remove the variable `coast` from the header: https://github.com/UBC-Thunderbots/Software/blob/e34155f9ca99ededa84077c75b00953d9d10dec1/src/software/ai/hl/stp/tactic/stop/stop_fsm.h#L21